example/src/bin/docs_examples.rs

```diff
@@ -157,16 +123,8 @@ pub async fn example_API_Increment(
     cache_client: &CacheClient,
     cache_name: &String,
 ) -> Result<(), MomentoError> {
-    match cache_client.increment(cache_name, "key", 1).await {
-        Ok(r) => println!("Incremented value: {}", r.value),
-        Err(e) => {
-            if let MomentoErrorCode::NotFoundError = e.error_code {
-                println!("Cache not found: {}", cache_name);
-            } else {
-                eprintln!("Error incrementing value in cache {}: {}", cache_name, e);
-            }
-        }
-    }
+    cache_client.increment(cache_name, "key", 1).await?;
+    println!("Value incremented");
```

Comment:
For this one the response has the new value, which is valuable to print. Can you update the print statement for that?